resources/views/auth/manager_registration_fields.html.twig

```diff
@@ -6,21 +6,21 @@
     It requires the /js/managerRegistration.js script in order to function properly.
 
 ============================================================================= #}
-{% set visibility = "hidden" %}
 <div data-c-grid-item="base(1of1)" data-c-input="select">
+	{% set visibility = "hidden" %}
 	<label for="department">
 		{{ copy.department }}
 	</label>
 	<span>{{ trans('forms.required') }}</span>
 	<div>
 		<i class="fa fa-caret-down"></i>
 		<select id="department" name="department" required>
-			<option selected value="" disabled>{{ copy.department_null_selection }}</option>
+			<option {% if user.department_id == null %} selected {% endif %} value="" disabled>{{ copy.department_null_selection }}</option>
 			{% if not_in_gov_option %}
 				<option value="{{ not_in_gov_option.value }}">{{ not_in_gov_option.name }}</option>
 			{% endif %}
 			{% for dept in departments %}
-				{% if old('department') == dept.id %}
+				{% if user.department_id == dept.id %}
```

Comment:
Curious as to why this got changed? It's supposed to grab the old input if validation fails, so users don't need to re-enter data.

Reply:
I was getting unwanted results with that, if the user had already selected a department. Just added or logic and seems to work fine.

Reply:
Oh true. Nice, I guess you could do the same for the `gov_email` to. 
`value="{{ user.gov_email ?: old('gov_email') }}"`

Reply:
Wait, shouldn't Managers and HR Advisors only be hitting the registration form once. Or else they'd be able to change their department.

Reply:
1. Done 👍 
2. They should be. More info on that in the issue description.